config/secrets.yml

```diff
@@ -0,0 +1,22 @@
+# Be sure to restart your server when you modify this file.
```

Comment:
Be careful to NEVER check this file into git. You should probably regenerate your keys and update your repo :closed_lock_with_key: